docs/uart.md

```diff
@@ -39,9 +43,8 @@ dictionary UARTOptions {
 
 [NoInterfaceObject]
 interface UARTConnection {
-    Promise<void> init(UARTOptions options);
     // void close();
-    Promise<void> write(Buffer data);
+    void write(Buffer data);
```

Comment:
But now UART just became a synchronous API. Is that warranted?
((The same question applied to the other changes as well).